lost/src/test/java/com/mapzen/android/lost/internal/LostApiClientImplTest.java

```diff
@@ -13,10 +17,6 @@
 import org.robolectric.RobolectricGradleTestRunner;
 import org.robolectric.annotation.Config;
 
-import android.content.ComponentName;
```

Comment:
Using the `MapzenAndroid` code style android imports should stay below mapzen and third party.